api/src/main/java/org/pmiops/workbench/api/WorkspacesController.java

```diff
@@ -435,6 +437,23 @@ public ResponseEntity<EmptyResponse> deleteWorkspace(
     return ResponseEntity.ok(new EmptyResponse());
   }
 
+  @Override
+  public ResponseEntity<String> getWorkspaceAccess(String workspaceNamespace) {
+    try {
+      DbWorkspace workspace = workspaceService.lookupWorkspaceByNamespace(workspaceNamespace);
```

Comment:
Need to check that this user has at least viewer access on this workspace to have access to this info. This will keep a logged in user from seeing other workspace access levels that they don't have access to.
```
workspaceAuthService.enforceWorkspaceAccessLevel(
        workspace.getWorkspaceNamespace(), workspace.getFirecloudName(), WorkspaceAccessLevel.VIEWER);
```